drivers/pwm/pwm_nrfx.c

```diff
@@ -126,8 +294,10 @@ static int pwm_nrfx_init(struct device *dev)
 static void pwm_nrfx_uninit(struct device *dev)
 {
 	const struct pwm_nrfx_config *pconfig = dev->config->config_info;
+	NRF_PWM_Type *pwm_instance = pconfig->pwm.p_registers;
 
-	nrfx_pwm_uninit(&pconfig->pwm);
```

Comment:
Why? The driver has to be deinitialized when a low power state is entered. Otherwise, the re-initialization when returning to the active state will fail.

Reply:
Agreed.